src/selectedtests/work_items/process_task_mapping_work_items.py

```diff
@@ -60,10 +61,10 @@ def _process_one_task_mapping_work_item(
     :param mongo: An instance of MongoWrapper.
     :param after_date: The date at which to start analyzing commits of the project.
     """
-    log = LOGGER.bind(project=work_item.project, module=work_item.module)
-    log.info("Starting task mapping work item processing for work_item")
-    if _seed_task_mappings_for_project(evg_api, mongo, work_item, after_date, log):
-        work_item.complete(mongo.task_mappings_queue())
+    with tmp_bind(LOGGER, project=work_item.project, evergreen_module=work_item.module) as log:
```

Comment:
Binding a `module` param raises a `KeyError: "Attempt to overwrite 'module' in LogRecord"` so I renamed it to `evergreen_module`